### DISCUSSION TITLE
Export game to .exe

### DISCUSSION THREAD

**Discussion by hk-hua-21** [o3de/o3de#16005] [Help Wanted] [answered]
Hey everyone! I need a little help. I have developed a simple game in O3DE and now want to export it to a .exe file just like we do in Unity. I am not sure if we can do it or not in O3DE yet and are there any tutorials/steps to do it? Thanks in advance :)

**Comment by cgalvan** [comment, marked as answer]
Hello! We do have steps to do this on Windows here: https://www.o3de.org/docs/user-guide/packaging/windows-release-builds/

There is an effort currently to streamline the process as outlined by this RFC: https://github.com/o3de/sig-content/blob/main/rfcs/rfc-123-project-export-cli.md

With the most recent related PR being: https://github.com/o3de/o3de/pull/15463

@tkothadev

**Comment by hk-hua-21** [comment]
Thank you so much!
